i18n/fr/docusaurus-plugin-content-docs/current.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Where is this file used? It's full of very generic text.

Reply:
This file comes when the command "npm write-translations" is run. This is a docusaurus command which helps in creating a folder that contains the translated files.
I will check if this is needed or not and then will / will not push it in the next commit as per need.